app/sections/newsletter/newsletter-form.tsx

```
@@ -51,14 +51,14 @@ const NewsLetterForm = forwardRef<HTMLDivElement, NewsLetterInputProps>(
         </Form>
         {helpText && (
           <div
-            className="text-body-subtle mt-2"
+            className="mt-2 text-body-subtle"
             data-motion="fade-up"
             dangerouslySetInnerHTML={{ __html: helpText }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)